src/navfeed.sol

```diff
@@ -81,24 +84,34 @@ contract NAVFeed is BaseNFTFeed, Interest, Buckets  {
     }
 
     /// Ceiling Implementation
-    function borrow(uint loan, uint amount) external auth {
+    function borrow(uint loan, uint amount) external auth returns(uint navIncrease) {
+        uint navIncrease = _borrow(loan, amount);
+        approximatedNAV = safeAdd(approximatedNAV, navIncrease);
+        return navIncrease;
```

Comment:
don't think we need to return anything here, would chenge the interface of the function